packages/client/src/pages/location/[locationName].tsx

```diff
@@ -30,6 +32,19 @@ const LocationPage = () => {
   useEffect(() => {
     dispatch(LocationAction.setLocationName(locationName))
     AuthService.listenForUserPatch()
+
```

Comment:
lets put these injected system in a helper function so we can easily modularise and extend this without having to duplicate changes in the offline route and any custom routes in projects that wish to use this too